extensions-core/protobuf-extensions/pom.xml

```diff
@@ -35,7 +35,7 @@
   </parent>
 
   <properties>
-    <commons-io.version>2.11.0</commons-io.version>
+    <commons-io.version>2.17.0</commons-io.version>
     <okio.version>3.6.0</okio.version>
```

Comment:
note: setting the version in this submodule have not much meaning - as its configured with `dependencyManagement` at the top level pom; this and the `version` for the package can be removed..

this `okio` thing seems to be an interesting thing as well

it seems we have 3 versions of that lib loaded; so it might be 
```
$ rebuild -Pdist -Pbundle-contrib-exts
$ tar tzf ./distribution/target/apache-druid-32.0.0-SNAPSHOT-bin.tar.gz |grep okio
apache-druid-32.0.0-SNAPSHOT/extensions/druid-iceberg-extensions/okio-2.8.0.jar
apache-druid-32.0.0-SNAPSHOT/extensions/druid-kubernetes-extensions/okio-jvm-3.6.0.jar
apache-druid-32.0.0-SNAPSHOT/extensions/druid-kubernetes-extensions/okio-3.6.0.jar
apache-druid-32.0.0-SNAPSHOT/extensions/druid-kubernetes-overlord-extensions/okio-1.15.0.jar
```

might worth a try to remove the version for that too and possibly declare its version via `dependencyManagement` in the root pom.xml in a separate pr

Reply:
you want me to remove commons io from submodule as part of this pr only or that too as another pr ?